site/source/index.md

```diff
@@ -0,0 +1,212 @@
+---
```

Comment:
Hm, so we will have `meteor/blaze/README.md` and `meteor/blaze/packages/blaze/README.md` (which will go to Atmosphere) and then this file. This feels to me like ...  a bit of duplication. How can we make sure that those all files are reasonable and in sync somehow?